csrc/mmdeploy/preprocess/transform/resize.cpp

```diff
@@ -23,8 +23,11 @@ ResizeImpl::ResizeImpl(const Value& args) : TransformImpl(args) {
         MMDEPLOY_ERROR("'size' expects an array of size 2, but got {}", args["size"].size());
         throw std::length_error("'size' expects an array of size 2");
       }
-      auto height = args["size"][0].get<int>();
-      auto width = args["size"][1].get<int>();
+      auto width = args["size"][0].get<int>();
```

Comment:
do not modifiy well documented behavior, as (h,w) tuple is a common pattern in data pipelines.